priv/perf/apps/load_test/lib/child_chain/deposit.ex

```diff
@@ -0,0 +1,78 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule LoadTest.ChildChain.Deposit do
+  @moduledoc """
+  Utility functions for deposits on a child chain
+  """
+  require Logger
+
+  alias ExPlasma.Encoding
+  alias ExPlasma.Transaction.Deposit
+  alias ExPlasma.Utxo
+  alias LoadTest.Ethereum
+  alias LoadTest.Ethereum.Account
+
+  @eth <<0::160>>
+  @poll_interval 5_000
+
+  def deposit_from(%Account{} = depositor, amount, currency, deposit_finality_margin) do
+    deposit_utxo = %Utxo{amount: amount, owner: depositor.addr, currency: currency}
+    {:ok, deposit} = Deposit.new(deposit_utxo)
+    {:ok, {deposit_blknum, eth_blknum}} = send_deposit(deposit, depositor, amount, currency)
+    :ok = wait_deposit_finality(eth_blknum, deposit_finality_margin)
+    Utxo.new(%{blknum: deposit_blknum, txindex: 0, oindex: 0})
+  end
+
+  defp send_deposit(deposit, account, value, @eth) do
+    eth_vault_address = Application.fetch_env!(:ex_plasma, :eth_vault_address)
```

Comment:
wait... where does ex plasma get it's vault address? ex_plasma should be completely stateless

Reply:
I would suggest we avoid this, ex_plasma will deprecate and remove this. it was a mistake to include it.

Reply:
I don't get what you mean here. We are not using `ExPlasma` for making deposits.

Reply:
why are we setting contract address to `ex_plasma`?

Reply:
I guess it's a leftover from the times before we moved away from depending on `:omg*` apps. Let me tidy up those and other settings. There are couple of issues with them.